packages/components/src/public/QueryModel/TabbedGridPanel.tsx

```diff
@@ -44,11 +47,16 @@ const GridTab: FC<GridTabProps> = memo(({ isActive, model, onSelect, pullRight,
     });
     const onClick = useCallback(() => onSelect(id), [id, onSelect]);
 
+    const rowCountDisplay = useMemo(() => {
+        if (rowCount === undefined && !(model.rowsLoadingState === LoadingState.LOADING)) return tabRowCount;
```

Comment:
note that I think we'll want to change this to check the new `model.totalCountLoadingState` after my changes from fb_gridAsyncTotalCount is merged

Reply:
@XingY FYI, my changes were just merged to develop and this `model.isLoadingTotalCount` should now be ready to update

Reply:
awesome. Updated.

Reply:
turns out model.isLoadingTotalCount doesn't work for this scenario. I added a isActivelyLoadingTotalCount to check if the queryModel is either actively loading row count or rows. For tabs with autoload = false, I do want the passed in count to display when autoload is false. So the checking of INITIALIZED won't work, and the checking of totalCountLoadingState is not sufficient since rowsLoadingState also might be updating row count.

Reply:
In the case of a QueryModel that is set to load the total count via `includeTotalCount` then the row count that comes back from the loadRows API call will not get set in the model, so the loading state shouldn't be related to `rowsLoadingState`.